_source/logzio_collections/_log-sources/aws-cross-account.md

```diff
@@ -0,0 +1,223 @@
+
+[![Deploy to AWS](https://dytvr9ot2sszz.cloudfront.net/logz-docs/lights/LightS-button.png)](https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=aws-cross-account-destination&templateURL=https://integrations-testing.s3.amazonaws.com/cross-accounts/0.0.1/sam-template-destination.yaml)
+
+
+##### Specify the stack template
+
+Keep the default settings in the **Create stack** screen and select **Next**.
+
+##### Specify the stack details
+
+Specify the stack details as per the table below and select **Next**.
+
+| Parameter          | Description                                                                                                                                                 |
+|--------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `RoleArn`          | The ARN of the **Role** created in the main stack. You can find it in the main stack's **Outputs** tab, under **LogzioCrossAccountRole**                        |
+| `KinesisStreamArn` | The ARN of the **Kinesis Stream** created in the main stack. You can find it in the main stack's **Outputs** tab, under **LogzioCrossAccountKinesisStreamArn**. |
+| `SendingAccounts`        | Comma-delimited list (**no spaces**) of account numbers of the accounts you'd like to send logs froms. The numbers should be specified as follows: `"1234","5678","9012"` | 
+
+##### Configure stack options
+
+If required, specify the **Key** and **Value** parameters for the **Tags** and select **Next**.
+
+##### Review the stack
+
+Confirm that you acknowledge that AWS CloudFormation might create IAM resources, IAM resources with custom names, may require `CAPABILITY_AUTO_EXPAND` and select **Create stack**.
+
+
+##### Create a subscription filter in your sending accounts
+
+<!-- info-box-start:info -->
+You need to create a subscription filter in each sending account separately.
+{:.info-box.note}
+<!-- info-box-end -->
+
+1. Make sure your AWS CLI is connected to the account you want to send logs from.
+2. Make sure you have set the CLI to the region of the account that you need to send logs from.
+3. Run the following command:
+
+```shell
+aws logs put-subscription-filter \
+    --log-group-name "<<LOG-GROUP-NAME>>" \
+    --filter-name "<<SUBSCRIPTION-FILTER-NAME>>" \
+    --filter-pattern " " \
+    --destination-arn "<<DESTINATION-ARN>>"
+```
+   * Replace `<<LOG-GROUP-NAME>>` with the name of the log group you want to collect logs from.
+   * Replace `<<SUBSCRIPTION-FILTER-NAME>>` with the name of the subscription filter you create.
+   * Replace `<<DESTINATION-ARN>>` with the ARN of the destination you created in the landing account. You can find the ARN in the main stack's **Outputs** tab. The destination will be in the same region as your log group. 
```

Comment:
Not sure how to phrase it, but the Destination Arn doesn't have to be the one from the main stack, but it should be the arn of the destination that matches the region that they want to ship their logs from. For example - if the log stream is in `us-west-2`, then they should use the arn of the Destination that's in `us-west-2`.